xarray/namedarray/parallelcompat.py

```diff
@@ -95,23 +101,32 @@ def guess_chunkmanager(
     Else use whatever is installed, defaulting to dask if there are multiple options.
     """
 
-    chunkmanagers = list_chunkmanagers()
+    available_chunkmanagers = list_chunkmanagers()
 
     if manager is None:
-        if len(chunkmanagers) == 1:
+        if len(available_chunkmanagers) == 1:
             # use the only option available
-            manager = next(iter(chunkmanagers.keys()))
+            manager = next(iter(available_chunkmanagers.keys()))
         else:
             # use the one in options (default dask)
             manager = OPTIONS["chunk_manager"]
 
     if isinstance(manager, str):
-        if manager not in chunkmanagers:
+        if manager not in available_chunkmanagers and manager in KNOWN_CHUNKMANAGERS:
+            raise ImportError(
+                f"chunk manager {manager!r} is not available."
+                f" Please make sure {KNOWN_CHUNKMANAGERS[manager]} is installed and importable."
+            )
+        elif len(available_chunkmanagers) == 0:
+            raise ImportError(
+                "no chunk managers available. Try installing `dask` or another package that provides a chunk manager."
+            )
+        elif manager not in available_chunkmanagers:
             raise ValueError(
-                f"unrecognized chunk manager {manager} - must be one of: {list(chunkmanagers)}"
+                f"unrecognized chunk manager {manager} - must be one of: {list(available_chunkmanagers)}"
```

Comment:
re-reading this, it looks like only this error message is only describing the issue, not offering advice. What do you think about extending it?
```suggestion
                f"unrecognized chunk manager {manager} - must be one of: {list(available_chunkmanagers)}"
                " (you can extend this list by installing more packages that provide a chunk manager)."
```

Reply:
Sounds good. You could also change it to say "must be one of the installed chunkmanagers: {{list(available_chunkmanagers)}}" which would be a bit clearer about what the list represents.